rust-toolchain.toml

```diff
@@ -0,0 +1,2 @@
+[toolchain]
+channel = "stable"
```

Comment:
is this file really necessary?

Reply:
I'm not sure. For a time, this library only compiles with 1.76 tool chain. Do we have requirements for compatibility?